appveyor.yml

```diff
@@ -18,6 +18,9 @@ environment:
 build_script:
   - cd ..
   - git clone --depth=1 https://github.com/xbmc/xbmc.git
+  - IF %GENERATOR% == "Visual Studio 14" (cd xbmc\tools\buildsteps\win32) ELSE (cd xbmc\tools\buildsteps\x64)
```

Comment:
`IF "%GENERATOR%" == "Visual Studio 14"`

Reply:
that fails in my win7 cmd with that stdio unexpected message. but let's try it..